.github/workflows/changelog.yml

```
@@ -40,6 +40,8 @@ jobs:
               - 'packages/scripts/**'
             voltoSlate:
               - 'packages/volto-slate/**'
+            documentation:
+              - 'packages/docs/**'
```

Please revert any relocation of docs. It is for all packages in the repo, and it is not a separate package unto itself. Moving it will also break the main documentation.

---

ah, this is wrong, i did not rlocate it, just forgot to remove the leading packagces.
```suggestion
              - 'docs/**'
```